cmd/goal/clerk.go

```diff
@@ -173,6 +174,7 @@ func init() {
 	simulateCmd.Flags().BoolVar(&simulateStackChange, "stack", false, "Report stack change during simulation time")
 	simulateCmd.Flags().BoolVar(&simulateScratchChange, "scratch", false, "Report scratch change during simulation time")
 	simulateCmd.Flags().BoolVar(&simulateAllowUnnamedResources, "allow-unnamed-resources", false, "Allow access to unnamed resources during simulation")
+	simulateCmd.Flags().BoolVar(&simulateInitialStateEnable, "initial-state-enable", false, "Return application byte codes executed during simulation")
```

Comment:
since we are shrinking the whole config to one command, then I changed the description for all initial states of resources accessed during simulation.